### PR TITLE
Auth back end

### PR DESCRIPTION
**Pull request by AlexRSc**
create Login functionality with auth-functionality

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -2,19 +2,24 @@ import {Route, BrowserRouter as Router, Switch} from "react-router-dom";
...
 
 export default function App() {
   return (
+      <AuthProvider>
       <Router>
           <Switch>
           <Route exact path="/"><WelcomePage/></Route>
           <Route path="/login"><LoginPage/></Route>
           <Route path="/registration"><RegistrationPage/></Route>
+          <Route path="/Home"><HomePage/></Route>
```
Comment (Droggelbecher92) [inline frontend/src/App.js]:
```suggestion
          <Route path="/home"><HomePage/></Route>
```

Comment (Droggelbecher92) [reply frontend/src/App.js]: should all be small-case for URL

File: frontend/src/App.js [hunk 2]
```diff
@@ -2,19 +2,24 @@ import {Route, BrowserRouter as Router, Switch} from "react-router-dom";
...
           <Route path="/Info"></Route>
```
Comment (Droggelbecher92) [inline frontend/src/App.js]:
```suggestion
          <Route path="/info"></Route>
```

Comment (Droggelbecher92) [reply frontend/src/App.js]: same

File: frontend/src/App.js (hunk 1 above)
Comment (AlexRSc) [reply frontend/src/App.js]: du hast recht 👍

File: frontend/src/pages/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import Footer from "../components/Footer";
+import styled from "styled-components"
+
+
+export default function HomePage() {
+    return (
+        <PageLayout>
+            <Header title="Home"/>
+            <Wrapper>
+            <h1>Home</h1>
```
Comment (Droggelbecher92) [inline frontend/src/pages/HomePage.js:12]: Is this a placeholder?

File: frontend/src/services/user-api-service.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export const getToken = credentials =>
+    axios.post('api/ProfBeluga/user/login', credentials)
```
Comment (Droggelbecher92) [inline frontend/src/services/user-api-service.js:4]: same, use small-case

**Review by Droggelbecher92 — APPROVED** [review]
This is fine, only minor stuff 👍

File: frontend/src/pages/HomePage.js (hunk 3 above)
Comment (AlexRSc) [reply frontend/src/pages/HomePage.js:12]: yep

**Pull request merged**